host_cleanup.sh

```diff
@@ -46,6 +46,12 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -i vm-setup/inventory.ini \
     -b -vvv vm-setup/teardown-playbook.yml
 
+ANSIBLE_FORCE_COLOR=true ansible-playbook \
+    -e "{use_firewalld: $USE_FIREWALLD}" \
```

Comment:
Is the intention here to lookup for` USE_FIREWALLD` variable that is set on the current shell session and inherit its value to `use_firewalld`? If yes, I think you don't need {}. See the example [here](https://github.com/metal3-io/metal3-dev-env/blob/0f21decd48128ba1df52385ec2f4edae2462a593/scripts/v1alphaX/provision_cluster.sh#L18)